### PR TITLE
src/osfv/cli/cli.py: fix rte rel tgl

### PR DESCRIPTION
**Pull request by pietrushnic**
Use predefined states in RTE class instead of string.

### CODE REVIEW

File: osfv_cli/src/osfv/cli/cli.py [hunk 1]
```diff
@@ -238,10 +238,10 @@ def print_asset_details_for_zabbix(asset):
 
 def relay_toggle(rte, args):
     state_str = rte.relay_get()
-    if state_str == "off":
-        new_state_str = "on"
+    if state_str == RTE.PSU_STATE_OFF:
```
Comment (macpijan) [inline osfv_cli/src/osfv/cli/cli.py:241]: Looks good - thanks

File: osfv_cli/src/osfv/cli/cli.py [hunk 2]
```diff
@@ -656,6 +656,9 @@ def main():
     check_out_group = check_out_parser.add_mutually_exclusive_group(
         required=True
     )
+    check_out_group = check_out_parser.add_mutually_exclusive_group(
```
Comment (macpijan) [inline osfv_cli/src/osfv/cli/cli.py:659]: Why is it needed?

File: osfv_cli/src/osfv/cli/cli.py [hunk 3]
```diff
@@ -689,6 +692,9 @@ def main():
     check_in_group = check_in_parser.add_mutually_exclusive_group(
         required=True
     )
+    check_in_group = check_in_parser.add_mutually_exclusive_group(
```
Comment (macpijan) [inline osfv_cli/src/osfv/cli/cli.py:695]: As above

File: osfv_cli/src/osfv/cli/cli.py (hunk 2 above)
Comment (macpijan) [reply osfv_cli/src/osfv/cli/cli.py:659]: Likely simply added by mistake?

Comment (macpijan) [reply osfv_cli/src/osfv/cli/cli.py:659]: I will assume so and will proceed with merging all of that, since I have already approved the rest of PRs.

Comment (pietrushnic) [reply osfv_cli/src/osfv/cli/cli.py]: @macpijan mistake during rebase on top of code style cleanup branch.

**Pull request merged**